Translation corrections (#90)
07th-mod/higurashi-console-arcs: Update/some_tips_082.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s20/28/vtt_tomoe3038", 256, TRUE);
	OutputLine(NULL, "「さすがに、もう上がったかもしれませんね…。",

> 			NULL, "\"He might already be done for the day at this time...", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"He might already be done for the day by now...", Line_WaitForInput);
```
